nucypher-core-wasm/src/lib.rs

```diff
@@ -347,6 +358,20 @@ impl TreasureMap {
         Ok(serde_wasm_bindgen::to_value(&result)?)
     }
 
+    #[wasm_bindgen(js_name = makeRevocationOrders)]
+    pub fn make_revocation_orders(&self, signer: &Signer) -> Vec<JsValue> {
```

Comment:
This should probably be made a method in the Rust part, and then exposed via bindings.